charts/stable/home-assistant/README.md

```diff
@@ -23,7 +23,7 @@ Kubernetes: `>=1.16.0-0`
 | https://charts.bitnami.com/bitnami | influxdb | 1.1.9 |
 | https://charts.bitnami.com/bitnami | mariadb | 9.3.6 |
 | https://charts.bitnami.com/bitnami | postgresql | 10.3.15 |
-| https://library-charts.k8s-at-home.com | common | 2.2.0 |
+| https://library-charts.k8s-at-home.com | common | 2.3.0 |
```

Comment:
Unsure why 2.3.1 wasn't picked up, you might need to re-run helm-docs

Reply:
I probably ran the README gen before I pushed the fix for 2.3.0. Will to a quick update 👍